README.md

```diff
@@ -123,6 +123,27 @@ use the `--distro <distro>:<version>` flag. A full example is:
 grype --add-cpes-if-none --distro alpine:3.10 sbom:some-apline-3.10.spdx.json
 ```
 
+### Scan attestations
+Grype can scan SBOMs from attestations, as long as they are encoded as part of (in-toto envelope)[https://github.com/in-toto/attestation/blob/main/spec/README.md#envelope], that is the format used by Syft.
```

Comment:
nit:
```suggestion
Grype can scan SBOMs from attestations, as long as they are encoded as part of and [in-toto envelope](https://github.com/in-toto/attestation/blob/main/spec/README.md#envelope).
```

This should clean up the link. Let me know if you think it reads better by dropping the end statement